src/model/selection.js

```diff
@@ -562,6 +562,50 @@ export default class Selection {
 	}
 
 	/**
+	 * Gets elements of type "block" touched by the selection.
+	 *
+	 * This method's result can be used for example to apply block styling to all
+	 * blocks covered by this selection.
+	 *
+	 * In this case the function will return all 3 paragraphs:
```

Comment:
What exactly will be returned? Just 3 paragraphs? Or 3 paragraphs and quote? Clarification needed. If just 3 paragraphs it need to be clarified that quote is not seen as block.